Dispatcher.php

```diff
@@ -76,8 +87,11 @@ public function parseTarget($target)
                 if (strpos($target, $currentSeparator) !== false) {
                     $separatorFound = true;
                     $explodedInfos = explode($currentSeparator, $target);
+
                     $this->controller = $explodedInfos[0];
                     $this->method = $explodedInfos[1];
+                    // Not sure this can be done
+                    $this->role = $explodedInfos[2];
```

Comment:
what if developer provides only two elements (controller & method) ?
and is this code compatible with current AltoRouter & AltoDispatcher use ? I think not.

Reply:
Like this its not working perfectly, but i tried.